### PR TITLE
Change branch name

### PR DESCRIPTION
**Pull request by garma-a**
change branch name done

### CODE REVIEW

File: front/src/components/UserForm.tsx [hunk 1]
```diff
@@ -1,11 +1,117 @@
...
+                      <p >{errors.birthDate?.message}</p>
+                     {/* Difficulty Level */}
+                     <select  {...DifficultyRegister()}>
+                            <option>Easy</option>
+                            <option>Medium</option>
+                            <option>Hard</option>
+                     </select>
+                      <p >{errors.level?.message}</p>
+                     {/* Main Message */}
+                     <textarea placeholder='Message' id='confirm' {...MessageRegister()} />
```
Comment (DevAbdoTolba) [inline front/src/components/UserForm.tsx]: `<input required... >`
`<input required... >`
`<select required... >`
`<textarea required... >`

- [x] make all inputs required

File: front/src/components/UserForm.tsx [hunk 2]
```diff
@@ -1,11 +1,117 @@
...
+ return (<>
+               <form onSubmit={handleSubmit(onSubmit)} >
+                     {/* Name Input Field */}
+                     <input type='text' id='name' placeholder='Name' {...NameRegister()} />
+                     <p>{ errors.username?.message}</p>
+                     {/* BithDate Input Field */}
+                     <input id='date' type='date' {...DateRegister()} />
+                      <p >{errors.birthDate?.message}</p>
+                     {/* Difficulty Level */}
+                     <select  {...DifficultyRegister()}>
```
Comment (DevAbdoTolba) [inline front/src/components/UserForm.tsx]: - [x] set a default value as None and make sure the user changes it

**Review by DevAbdoTolba — CHANGES_REQUESTED** [review]
Great good 🤩. small changes required

**Comment by garma-a** [comment]
Effectively implemented All changes

**Review by DevAbdoTolba — APPROVED** [review]
LGTM 👍

**Pull request merged**